### ISSUE TITLE
[Wii] Slight hang when saving in Pokemon

### ISSUE THREAD

**Issue by SuperrSonic** [mgba-emu/mgba#433]
When saving, the game audio buzzes and hangs for a second. This only seems to happen on the first save until the next fresh boot.

Tested Pokemon Emerald and Ruby.

**Comment by endrift** [comment]
This is because it's taking time to actually copy the save to disk. I'd have to do async disk I/O to fix this, which A) I'm not even sure is possible on the Wii (I guess I could use the lwt system?) and B) would require changes to the VFile API.

**Comment by SuperrSonic** [comment]
I see, I thought it was odd that it would only happen on the first save but I just tried a faster SD card(although with less space) and it was hardly noticeable.

Should I close this?

**Comment by endrift** [comment]
No, please leave it open so I can try to figure out a better way to implement this.
